data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,31 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4105508"
+    name = "pr_cianorte"
+    allowed_domains = ["cianorte.pr.gov.br"]
+    start_urls = ["https://cianorte.pr.gov.br/orgao-oficial/"]
+    current_page = 1
+    base_url = "https://cianorte.pr.gov.br/orgao-oficial/pagina/"
+
+    def parse(self, response):
+        lines = response.css("tr")[1:]
+        if len(lines):
+            for row in lines:
```

Comment:
Just a common minor improvement to gain horizontal space:
```suggestion
        lines = response.css("tr")[1:]
        if not lines:
        	return
        	
        for row in lines:
        ...
```